crate/operator/constants.py

```diff
@@ -24,6 +24,7 @@
 LABEL_NAME = "app.kubernetes.io/name"
 LABEL_PART_OF = "app.kubernetes.io/part-of"
 LABEL_NODE_NAME = f"{API_GROUP}/node-name"
+LABEL_USER_PASSWORD = f"{API_GROUP}/user-password"
```

Comment:
Looking at [a recent change I made](0e2bbd44a37c32d0c1e4eb5e613b2a7290c8ad79), this should probably be
```suggestion
LABEL_USER_PASSWORD = f"operator.{API_GROUP}/user-password"
```